parity/run.rs

```diff
@@ -464,6 +464,12 @@ fn execute_impl<Cr, Rr>(cmd: RunCmd, logger: Arc<RotatingLogger>, on_client_rq:
 
 	let passwords = passwords_from_files(&cmd.acc_conf.password_files)?;
 
+	// run in daemon mode
```

Comment:
Maybe let's add a small comment here regarding moving the `daemonize` call, just so we don't make the same mistake in the future.